Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py

```diff
@@ -274,14 +275,15 @@
         )
     )
 
-# from Arnab via Martina 10/03/2016
+# LooseIDMVA [-0.9] SF and uncertainty for UL2017. Dt: 17/11/2020
+#link to the presentation: https://indico.cern.ch/event/963617/contributions/4103623/attachments/2141570/3608645/Zee_Validation_UL2017_Update_09112020_Prasant.pdf
 looseMvaBins = cms.PSet(
     variables = cms.vstring("abs(superCluster.eta)","full5x5_r9"),
     bins = cms.VPSet(
-        cms.PSet( lowBounds = cms.vdouble( 0.0, 0.0  ) , upBounds = cms.vdouble( 1.5, 0.85  ) , values = cms.vdouble( 0.9999 ) , uncertainties = cms.vdouble( 0.0001 )  ) ,
-        cms.PSet( lowBounds = cms.vdouble( 0.0, 0.85 ) , upBounds = cms.vdouble( 1.5, 999.0 ) , values = cms.vdouble( 1.0003 ) , uncertainties = cms.vdouble( 0.0000 )  ) ,
-        cms.PSet( lowBounds = cms.vdouble( 1.5, 0.0  ) , upBounds = cms.vdouble( 6.0, 0.9   ) , values = cms.vdouble( 1.0003 ) , uncertainties = cms.vdouble( 0.0000 )  ) ,
-        cms.PSet( lowBounds = cms.vdouble( 1.5, 0.9  ) , upBounds = cms.vdouble( 6.0, 999.0 ) , values = cms.vdouble( 1.0004 ) , uncertainties = cms.vdouble( 0.0000 ) ) ) ) 
+        cms.PSet( lowBounds = cms.vdouble( 0.0, 0.0  ) , upBounds = cms.vdouble( 1.5, 0.85  ) , values = cms.vdouble( 1.0021 ) , uncertainties = cms.vdouble( 0.0014 )  ) ,
+        cms.PSet( lowBounds = cms.vdouble( 0.0, 0.85 ) , upBounds = cms.vdouble( 1.5, 999.0 ) , values = cms.vdouble( 1.0001 ) , uncertainties = cms.vdouble( 0.0038 )  ) ,
+        cms.PSet( lowBounds = cms.vdouble( 1.5, 0.0  ) , upBounds = cms.vdouble( 6.0, 0.9   ) , values = cms.vdouble( 1.0061 ) , uncertainties = cms.vdouble( 0.0014 )  ) ,
```

Comment:
Hi @rchatter , the uncertainty 0.0014 in this line is different from the slide which is 0.0013. Is this an accuracy issue?

Reply:
@prasantrout could you please take a look at the comment above?

Reply:
Hii,
  
   Yes, actually the systematic quoted in the file is the quadrature sum of statistical uncertainty and the total systematic uncertaity on the scale factor.  Statistical uncertainty is generally very small compared to the systematic uncertainty.  For that low R9 bin in the endcap,  the systematic is found to be,  S =  sqrt(0.0004**2 + 0.0014**2) = 0.0014.    For all the other (eta, R9) bins, it is the same as the total systematic uncertainty in the scale factor.

Reply:
Sorry, My mistake. S = sqrt(S1*S2 + S2*S2), S1 = 0.0004 and S2 = 0.0014

Reply:
S1 = 0.0004, S2 = 0.0013, S = 0.0014